packages/myst-transforms/src/enumerate.ts

```diff
@@ -279,8 +296,13 @@ export class ReferenceState implements IReferenceState {
         }
       }
     });
+    this.identifiers = opts?.identifiers ?? [];
```

Comment:
`identifiers` are only added when `ReferenceState` is instantiated. This could change if we ever need another way to add these.